docs/topics/api-reference/http-api.md

```diff
@@ -45,75 +56,179 @@ Publishes a message to a topic.
+
+#### Headers
+
+| Header&nbsp;name | Required? | Description                                                                                                          |
+|------------------|-----------|----------------------------------------------------------------------------------------------------------------------|
+| `Authorization`  | no**      | The Momento auth token, in string format, is used for authentication/authorization of the request.                   |
+
+#### Responses
+
+### Success
+
+*Status code: 200 OK*
+
+- The subscription was successful and messages were returned. The response will contain a JSON object with the message(s) and their sequence numbers. If there was a gap in the message sequence, a discontinuity message will be included.
+
+### Error
+
+*Status code: 401 Unauthorized*
+- This error type typically indicates that the Momento auth token passed in is either invalid or expired. See the body of the message for further details.
+
+*Status code: 403 Forbidden*
+- This error type typically indicates the Momento auth token passed in does not grant the required access to the resources you attempted. See the body of the message for further details.
+
+*Status code: 404 Not Found*
+- “Cache Not Found” indicates that the specified cache or topic does not exist.
+
+*Status code: 429 Too Many Requests*
+- This error type typically indicates that account limits were exceeded. See the message body for further details, or contact Momento support to request a limit increase.
+
+*Status code: 500 Internal Server Error*
+- This error type typically indicates that the service is experiencing issues. Contact Momento support for further assistance.
+
+### Examples
+
+Below is an example that subscribes to the *example* topic in the *my-cache* cache in the *us-east-1* region, using a sequence number of `100` to ensure message continuity.
+
+#### Request with a sequence number provided
+
+This request includes auth as a query parameter.
+
+`curl -X GET "https://api.cache.cell-us-east-1-1.prod.a.momentohq.com/topics/my-cache/my-topic?token=<token>&sequence_number=100"`
+
+**Response**
+
+```json
+{
+  "items": [
+    {
+      "discontinuity": {
+        "last_topic_sequence": 100,
+        "new_topic_sequence": 104
+      }
+    },
+    {
+      "item": {
+        "topic_sequence_number": 104,
+        "value": {
+          "text": "hello world"
+        }
+      }
+    }
+  ]
+}
+
+```
+
+#### Request with no sequence number
+
+This request includes auth as a header.
+
+`curl -X GET -H "Authorization: <token>" "https://api.cache.cell-us-east-1-1.prod.a.momentohq.com/topics/my-cache/my-topic"`
+
+**Response**
+
+```json
+{
+  "items": [
+    {
+      "item": {
+        "topic_sequence_number": 0,
+        "value": {
+          "text": "hello world"
+        }
+      }
+    }
+  ]
+}
+```
```

Comment:
Just a nit but maybe put the simpler example w/ no sequence number specified first? Feels bit odd to put the simpler example below a more compicated example passing sequence number and getting discontinuity back?